armi/nuclearDataIO/cccc/gamiso.py

```diff
@@ -77,27 +77,33 @@ def addDummyNuclidesToLibrary(lib, dummyNuclides):
     if not dummyNuclides:
         runLog.important("No dummy nuclide data provided to be added to {}".format(lib))
         return False
-    if len(lib.xsIDs) > 1:
+    elif len(lib.xsIDs) > 1:
         runLog.warning(
             "Cannot add dummy nuclide data to GAMISO library {} containing data for more than 1 XS ID.".format(
                 lib
             )
         )
         return False
+
     dummyNuclideKeysAddedToLibrary = []
     for dummyNuclide in dummyNuclides:
-        dummyKey = dummyNuclide.nucLabel + lib.xsIDs[0]
+        # dummyKey = dummyNuclide.nucLabel + lib.xsIDs[0]
```

Comment:
delete commented code

Reply:
OMG. 

How many times have I told people that?  Soooo embarassing. No one saw that!